src/Test/Utilities/Portable/TestHelpers.cs

```diff
@@ -76,5 +79,59 @@ public static string AsXmlCommentText(string text)
             return result;
         }
 
+        #region Diagnostics
```

Comment:
Prefer to not use regions.

Reply:
This code is being moved over directly from `TestBase.cs`. I am moving it just to a static helper class so IDE tests can use a `using static = ...`

Reply:
Move from a type with regions to one without them.